lib/portlayer/exec/container_cache.go

```diff
@@ -99,11 +102,25 @@ func (conCache *containerCache) Remove(idOrRef string) {
 		delete(conCache.cache, container.ExecConfig.ID)
 		delete(conCache.cache, container.vm.Reference().String())
 	}
+}
+
+func (conCache *containerCache) sync(ctx context.Context, sess *session.Session) error {
+	cons, err := infraContainers(ctx, sess)
+	if err != nil {
+		return err
+	}
+
+	conCache.m.Lock()
+	defer conCache.m.Unlock()
+
+	conCache.cache = make(map[string]*Container)
```

Comment:
`sync` is called from Init, as is NewContainerCache().  Can this be moved to NewContainerCache() or a comment added as to why this needs to be here?